python/tvm/relay/frontend/onnx.py

```diff
@@ -2142,58 +2142,72 @@ class LSTM(RNN):
     """Operator converter for LSTM"""
 
     @classmethod
-    def generate_lstm(
-        cls, X_steps, H_t, C_t, W, R, B, p_i, p_f, p_o, f_act, g_act, h_act, backwards=False
-    ):
-        """Create an unrolled lstm loop.
-
-        See https://github.com/onnx/onnx/blob/master/docs/Operators.md for math.
+    def unbind(cls, data, axis=0):
```

Comment:
Move this to frontend/common.py if possible

Reply:
Should I do the same for pytorch frontend?  https://github.com/apache/tvm/blob/09e234dfd9dd8adf77cf4b6c39a371dd74530e14/python/tvm/relay/frontend/pytorch.py#L2095
More over just now I'm trying to implement this method (unbind) as relay operation, I follow to split op for this. But I think it will be separated PR

Reply:
done